packages/client/hmi-client/src/services/code.ts

```diff
@@ -118,10 +109,6 @@ async function uploadCodeToProjectFromGithub(
 		return null;
 	}
 
-	const resp = addAsset(projectId, AssetType.Code, newCode.id);
```

Comment:
I think you removed a lot of those by making the assumptions that we want to always save an asset to a project, which might not be the case in the future.

Reply:
I removed this addAsset call because I wasn't sure if `code.ts` should be doing `useProjects().addAsset` inside `uploadCodeToProjectFromGithub`, or whether the caller of `uploadCodeToProjectFromGithub` should subsequently call `useProjects().addAsset`.

e.g.

```
CodeService.uploadCodeToProjectFromGithub() //method implicitly adds the asset
```
vs.

```
CodeService.uploadCodeToProjectFromGithub() //doesn't add the asset
useProjects().addAsset() 
```